packages/react/mixins/mixin.browser.js

```diff
@@ -26,14 +26,12 @@ class ReactMixin extends Mixin {
     return createElement(BrowserRouter, props, element);
   }
   render() {
-    const { name } = this.config;
-    const attribute = `data-${name}`;
-    const mountpoint = document.getElementById(name);
-    const isMounted = mountpoint.hasAttribute(attribute);
+    const mountpoint = document.body.firstElementChild;
```

Comment:
This imho not specific enough. Can we be sure 100% of the time, that there's no Google TagManager(-like) script on the page, that inserts elements in a way, that corrupts our way of getting the mountpoint node?

Reply:
I agree. What do you propose instead? I would not like to rely on magic strings here - and the previous solution was similarly brittle.

Reply:
How about a `getMountpoint` hook?

Reply:
A dedicated hook for that feels a little too much to me. Wouldn't an adequately specific attribue name like `data-application-root` do the trick?

```js
const mountpoint = document.querySelector('[data-application-root]')
```

Besides: we could (maybe should) leave the old ID-attribute for peeps that — for whatever reason — rely on it in their code?!

Reply:
I like the `document.querySelector` approach - will update this PR accordingly.

The previous solution used to rely on `config.name` client-side while allowing people to alter it using `getTemplateData` server-side, making it very *surprising* to use. Besides, I fail to come up with valid use cases for that `id`...

Reply:
> Besides, I fail to come up with valid use cases for that `id`...

Other people might be more creative in that matter. But I guess after all this `id` is one of the internals of Untool & no user should rely on its existence. 🤷‍♂️

Reply:
Done - PTAL, @herschel666